### ISSUE TITLE
Need support for labels on tabs

### ISSUE THREAD

**Issue by aradhikaIbm** [IBM/carbon-components-angular#654]
## Detailed description

In trying to use the tab component, I need a way to add labels to the tab nav element. When there are multiple navigational elements in a page. A unique label is required. 

From DAP Chrome tool: 

https://aat.w3ibm.mybluemix.net/token/fc6655fc-6799-47a2-be20-9a6907013539/6dda174d-0092-4854-87be-d748c891b788/latest///doc/w3/help/en-US/idhi_accessibility_check_g1164.html

Currently using: 3.12.1

**Comment by youda97** [comment]
I ran the DAP tool in Chrome and I didn't see this violation appear. Can you share the code you are using and a screenshot of the DAP violation in chrome?

**Comment by aradhikaIbm** [comment]
Let me put something together and post a link by EOD. It will only appear if you more than one navigational item in your page.

**Comment by aradhikaIbm** [comment]
Sandbox: https://stackblitz.com/edit/carbon-angular-tabs?file=package.json

![image](https://user-images.githubusercontent.com/22798731/61464838-f2546c00-a944-11e9-8c5d-36fbf6aa3e3e.png)
